src/site/design-elements/buttons/index.html

```diff
@@ -1,23 +1,27 @@
 <div id="buttons" class="headroom-large">
-  <h5>Buttons</h5>
+  <h2>Buttons</h2>
   <p>This is the primary button style for important calls to action. Two primary buttons cannot be used next to each other.</p>
   <div data-xrayhtml>
-    <button class="btn btn-mapzen">main call to action</button>
+    <button class="btn btn-darkpurple">primary button</button>
```

Comment:
I prefer a `btn-primary` / `btn-secondary` type of naming scheme.  But I also know that can be more confusing than, say, `btn-white` when you know you just want a white button.

@louh @hanbyul-here - what are your thoughts on naming?